README.md

```diff
@@ -4,8 +4,8 @@ platforms: dotnet
 author: jmprieur
 level: 200
 client: Mobile (UWP)
-service: ASP.NET Web API
-endpoint: AAD V1
+service: ASP .NET Web API
+endpoint: Azure AD v1.0
 ---
 # Integrating a Windows Universal application with Azure AD

```

Comment:
Integrating a Windows Universal application with the **Microsoft identity platform** ?